client/src/components/_common/Message/Message.module.scss

```diff
@@ -57,6 +57,7 @@ p.is-scope-section {
 
   -webkit-appearance: none;
   appearance: none;
+  color: #222222;
```

Comment:
We should consider using CSS variables when available.* Global CSS variables are available in [`styles/settings`](https://github.com/TACC/Core-Portal/blob/main/client/src/styles/settings/). Please check whether [`--global-color-primary--xx-dark`](https://github.com/TACC/Core-Portal/blob/main/client/src/styles/settings/color.css#L30) works.

<details><summary>Footnotes</summary>

\* These variables allow us to manage colors, borders, _et cetera_ from a single location, which leads to a future where one stakeholders Portal could be styled differently (for unique branding) by only changing variables.

</details>